### PR TITLE
chore(sdk-curie): fix version, add targetPrice helper

### PR DESCRIPTION
**Pull request by sjillen**
I had no idea where to put that helper so it ended up as a static method in client. Open to any suggestion in that regard.
Fixing the version for sdk-curie allow the PerpWrapper to correctly fetch the price on Kovan Optimism

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -105,3 +105,4 @@ dist
 
 .npmrc
 .vscode
+.prettierrc
```
Comment (0xCryptoSheik) [inline .gitignore]: remove

File: src/lib/client.ts [hunk 2]
```diff
@@ -23,6 +23,10 @@ export class UXDClient {
     });
   }
 
+  public static calculateTargetPrice(price: number, slippage: number): number {
```
Comment (0xCryptoSheik) [inline src/lib/client.ts]: export function from a `utils/price.ts` file instead ?

File: src/lib/client.ts [hunk 3]
```diff
@@ -23,6 +23,10 @@ export class UXDClient {
...
+    return (price * (1000 - slippage * 10)) / 1000;
```
Comment (0xCryptoSheik) [inline src/lib/client.ts]: slippage is in the form of a percentage? `0.5` etc ?

Comment (sjillen) [reply src/lib/client.ts]: correct

**Pull request merged**